lib/controllers/mongo.js

```diff
@@ -23,6 +24,24 @@ Mongo.prototype.create = function (name, creatorId, regionData, callback) {
 }
 
 // -----------------------------------------------------------------------------
+Mongo.prototype.clone = function (db, hosts, callback) {
+  var api = userConfig.data.api_uri.split('://')
+
+  var Client = apiUtil({
+    protocol: api[0],
+    hostname: api[1],
+    port: api[0] === 'https' ? 443 : 80,
+    version: 'v2',
+    token: null
```

Comment:
same comment as above. I assumed this was for an admin token, but if putting the apiKey here prevents the need to pass in that second param, I can include it